Integrations/Tanium_v2/Tanium_v2.py

```diff
@@ -142,6 +152,80 @@ def update_id(self, obj):
             return obj
         return obj
 
+    def build_create_action_body(self, by_host, action_name, package_id, package_name,
```

Comment:
use default values to avoid passing empty strings to the function.
add unit test for the function